pkg/apis/core/types_shoot.go

```diff
@@ -186,6 +186,10 @@ type NetworkingStatus struct {
 	Nodes []string
 	// Services are the CIDRs of the service network.
 	Services []string
+	// Egress are the CIDRs used by the shoot as the source IP for egress traffic as reported by the used Infrastructure
+	// extension controller. For certain environments the egress IPs may not be stable in which case the extension
+	// controller may opt to not populate this field.
+	Egress []string
```

Comment:
I would have expected a name in plural similar to the other attributes. `Egresses` sounds somewhat weird to me, but I am no native speaker. `EgressCIDRs` or something similar would probably also be fine as long it does not imply a single value, but have a type of multiple values.

Reply:
`EgressCIDRs` works for me. It also matches the `Infrastructure.status.egressCIDRs` naming, so let's go with this.